action-server/alembic/versions/00003_fix_test_profiles.py

```diff
@@ -0,0 +1,64 @@
+"""Fix test profiles.
+
+Revision ID: 00003
+Revises:
+Create Date: 2020-05-14
+
+"""
+
+import structlog
+
+from alembic import op
+
+log = structlog.get_logger()
+
+revision = "00003"
+down_revision = "00002"
+branch_labels = None
+depends_on = None
+
+REMINDER_TABLE = "reminder"
+ASSESSMENT_TABLE = "assessment"
+
+OLD_TEST_PROFILE_REVISION = "rev_00002"
+NEW_TEST_PROFILE_REVISION = "rev_00003"
+
+
+def upgrade():
+    op.execute(
+        (
+            f"update reminder set attributes = attributes"
```

Comment:
Like discussed, see if we can use sqlalchemy operations instead

Reply:
as re-discussed, not really necessary for migration